src/ducks/SANCharts/Charts.js

```diff
@@ -43,18 +40,13 @@ import { checkHasPremium } from '../../pages/UserSelectors'
 import displayPaywall, { MOVE_CLB, CHECK_CLB } from './Paywall'
 import { binarySearch } from '../../pages/Trends/utils'
 import ChartWatermark from './ChartWatermark'
-import {
-  createTrigger,
-  fetchSignals,
-  removeTrigger
-} from '../Signals/common/actions'
-import { buildPriceSignal } from '../Signals/utils/utils'
 import SignalLine, {
   SignalPointSvg
 } from './components/newSignalLine/SignalLine'
 import SidecarExplanationTooltip from './SidecarExplanationTooltip'
 import sharedStyles from './ChartPage.module.scss'
 import styles from './Chart.module.scss'
+import WithSignals from './components/signalsChart/SignalsWrapper'
```

Comment:
Why there is a `components` folder inside `SANCharts` duck-module? Everything there is already **component**. Tightly related components should be putted inside common component folder, e.g. `Signal`. Then, there will be consistency for folder names: `ducks/SANCharts/_.js`, `/ducks/SANCharts/Signal/withSignals.js`, `/ducks/SANCharts/Signal/SignalLine.js`